tests/unit/utils/test_juju_utils.py

```diff
@@ -153,6 +153,20 @@ def test_coumodel_init(mocker):
     assert model.name == name
 
 
+@pytest.mark.asyncio
+async def test_coumodel_create(mocked_model):
+    """Test COUModel create function with no model_name defined."""
+    model = await juju_utils.COUModel.create(None)
```

Comment:
Can we also test model creation with name input? Parameterize it with `None` should be enough

Reply:
This is tested in `test_coumodel_init`, so that's why I skipped it. Do you think it's really needed?

Reply:
This looks good. It was hidden from the changes so I wasn't aware it was already tested elsewhere.